docs/changes.rst

```diff
@@ -208,7 +208,7 @@ Released: 2022-05-17
   (issues #950, #986)
 
 * Renamed the properties of the 'zhmcclient.testutils.HMCDefinition' to remove
-  the 'hmc_' prefix, e.g. 'hmc_userid' became 'userid', etc. (part of issue #986)
+  the 'hmc&nbsp;_' prefix, e.g. 'hmc_userid' became 'userid', etc. (part of issue #986)
```

Comment:
&nbsp; is on purpose, or is that a typo?

Reply:
That is on purpose. `&nbsp;` is the HTML symbol for non-breaking space which means it does not create any visible character but represents a character for the parsing. So the result looks like `'hmc_'` but it is apparently no longer parsed by RST as a link (see the builddoc item in https://github.com/zhmcclient/python-zhmcclient/actions/runs/4044091687/jobs/6953821628)

Actually, if I think about it, I don't know why it would no longer do that, but it seems to work.